commit f6b34fac68ff84b35ba8e85eb6440a1ea4b1eb89
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-08-15T16:27:15-04:00 (8 weeks ago)
    
    MANTA-4508 node-mahi provider transitive dependencies too old to build on x86_64

diff --git a/package.json b/package.json
index 9a70b22..8fd6296 100644
--- a/package.json
+++ b/package.json
@@ -1,16 +1,16 @@
 {
     "name": "mahi",
     "description": "Mahi client",
-    "version": "2.3.1",
+    "version": "2.3.2",
     "author": "Joyent (joyent.com)",
     "main": "index.js",
     "dependencies": {
         "aperture": "git://github.com/joyent/node-aperture.git#master",
         "assert-plus": "0.1.5",
-        "bunyan": "1.8.1",
+        "bunyan": "1.8.12",
         "http-signature": "0.10.0",
         "lru-cache": "4.1.5",
-        "restify": "4.1.1",
+        "restify": "4.3.4",
         "verror": "1.3.7"
     },
     "devDependencies": {
